crates/net/network/src/peers.rs

```diff
@@ -1896,6 +1902,64 @@ mod tests {
         assert!(!peers.peers.contains_key(&peer));
     }
 
+    #[tokio::test]
+    async fn test_fatal_outgoing_connection_error_trusted() {
+        let peer = PeerId::random();
```

Comment:
```suggestion
        let trusted_peer = PeerId::random();
```
nitpick, helps to follow along a bit better